packages/dd-trace/src/config.js

```diff
@@ -501,6 +504,7 @@ class Config {
     this._setValue(defaults, 'isManualApiEnabled', false)
     this._setValue(defaults, 'ciVisibilityTestSessionName', '')
     this._setValue(defaults, 'ciVisAgentlessLogSubmissionEnabled', false)
+    this._setValue(defaults, 'legacyBaggageEnabled', true)
```

Comment:
This being configured in a different way than the new baggage is confusing (one having a toggle and the other one being in the list of propagators)

Reply:
We are phasing out OpenTracing baggage so we don't want to add it as a supported value to `DD_TRACE_PROPAGATION_STYLE`.

Also, if we add a value for the OpenTracing baggage propagator (`ot-baggage`?) to `DD_TRACE_PROPAGATION_STYLE`, it would have to be included in the default list for Node.js, since OpenTracing is already enabled by default. But this would still be a breaking change for all users that _already_ set `DD_TRACE_PROPAGATION_STYLE`, since _none_ of them will have `ot-baggage` in their config.

By having a separate setting, we avoid a breaking change for users since we can keep it enabled by default for everyone (for now).

Reply:
My assumption was that the other config wouldn't be added to the propagators (since that's semantically incorrect) but if we are just following OpenTelemetry then I guess the discussion should happen over there.